core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -78,7 +78,7 @@ private[spark] class ExecutorAllocationManager(
 
   // How long there must be backlogged tasks for before an addition is triggered
```

Comment:
nit: Maybe mention in the comment that this is measured in seconds - not clear from the config name.